tests/test_api.py

```diff
@@ -3,8 +3,9 @@
 from n26 import api, config
```

Comment:
I changed nothing in this file except formatting (autoformat of PyCharm).
If you don't want that just let me know.